specifyweb/frontend/js_src/lib/components/QueryBuilder/Formatter.tsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 
 import { usePromise } from '../../hooks/useAsyncState';
 import { commonText } from '../../localization/common';
+import { queryText } from '../../localization/query';
+import { Button } from '../Atoms/Button';
 import { Select } from '../Atoms/Form';
+import { icons } from '../Atoms/Icons';
 import type { Tables } from '../DataModel/types';
 import { fetchFormatters } from '../Formatters/formatters';
-import { mappingElementDivider } from '../WbPlanView/LineComponents';
 
 export function QueryFieldFormatter({
```

Comment:
huh, appears like this component was added in https://github.com/specify/specify7/commit/d52aee94a01d126b2ef83f0dea1f690bb59f6ee1, but then code got lost?
because I don't see any usages of this component in the code anymore(

Reply:
yes, that's what I noted in the description of this pr